apps/web/src/pages/DirectivePage/Resources.tsx

```diff
@@ -163,68 +70,36 @@ const Resources = () => {
                 <p data-h2-padding="base(0 x1)">
                   {intl.formatMessage({
                     defaultMessage:
-                      "These resources are designed to support digital initiative managers and leads in fulfilling their responsibilities under the Directive on Digital Talent. This implementation guidance explains why the directive is needed and what is required. Use the decision tree to navigate the reporting requirements.",
-                    id: "Iz8E+y",
+                      "This implementation guidance is designed to support digital initiative managers and leads in fulfilling their responsibilities under the <cite>Directive on Digital Talent</cite>. It explains why the directive is needed and what is required.",
+                    id: "vYi5/y",
                     description:
                       "Body message for digital initiative managers section.",
                   })}
                 </p>
               </div>
               <div data-h2-padding="base(x1)">
-                <p data-h2-font-weight="base(bold)">
-                  {intl.formatMessage({
-                    defaultMessage: "Implementation guidance for managers",
-                    id: "wJ9hiY",
-                    description:
-                      "label above download guidance for managers pdf and text.",
-                  })}
-                </p>
                 <Link
-                  mode="inline"
+                  mode="text"
+                  data-h2-font-weight="base(bold)"
```

Comment:
Visually I prefer this change but was there a reason for it? It seems it now looks a bit different than the design.

## Design

![2024-09-18_09-53](https://github.com/user-attachments/assets/56e7c451-1179-495d-812f-74ac8a836100)

## Actual

![2024-09-18_09-53_1](https://github.com/user-attachments/assets/0728651d-221e-48fe-b4c0-db2151fd2616)

Reply:
Yeah I did this because the inline mode wrapped very strangely.
![image](https://github.com/user-attachments/assets/7b64b43c-0562-4cc9-ac8f-d6bea75798bd)